.github/workflows/ci.yaml

```diff
@@ -5,6 +5,7 @@ on:
   push:
     branches:
       - master
+      - 'release-*'
```

Comment:
I think we may need to run these things upon tag creation, too? Not sure how to convert the OSCI release process into GHA release